bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -73,10 +72,10 @@ export class UIEvents
     else if et == "move"
       logger.debug("Registering tool: #{type} for event '#{et}'")
       if tool_view._move_enter?
-        tool_view.listenTo(@, "move:enter", tool_view._move_enter)
-      tool_view.listenTo(@, "move", tool_view["_move"])
+        tool_view.listenTo(@, "move:enter:#{id}", tool_view._move_enter)
```

Comment:
I don't quite understand what you're asking. Are you saying that something that a listener on "move:enter" will also respond agnostically to "move:enter:id-hash"?